lib/pure/net.nim

```diff
@@ -365,8 +365,22 @@ proc bindAddr*(socket: Socket, port = Port(0), address = "") {.
   ## Binds ``address``:``port`` to the socket.
   ##
   ## If ``address`` is "" then ADDR_ANY will be bound.
-
-  if address == "":
+  ## If ``domain`` is AF_UNIX then ``address`` is the filepath.
+  if socket.domain == AF_UNIX:
```

Comment:
You really need to check for ``useWinVersion`` here. Otherwise on Windows you will get an ``undeclared identifier: 'sockaddr_un'`` error.